rules/cross-platform/threat_intel_indicator_match.toml

```
@@ -0,0 +1,183 @@
+[metadata]
+creation_date = "2021/04/21"
+maturity = "production"
+updated_date = "2021/10/29"
+min_stack_comments = "Threat intel module fields were updated from `threatintel.*` to `threat.*` in ECS 1.11 (7.16)."
+min_stack_version = "7.16"
```

If we don't want this change to go to pre-8.0 stacks

```suggestion
min_stack_version = "8.0"
```